src/address_to_census_tract.py

```diff
@@ -0,0 +1,374 @@
+#!/usr/bin/env python3
+"""
+Takes a file containing a list of addresses, one per line, and returns their
+associated census tract within Washington.
+
+To run:
+    ./src/address_to_census_tract.py {filepath} {institution}  # TODO 
+
+Examples:
+    ./src/address_to_census_tract.py data/test/testset.json
+    ./src/address_to_census_tract.py data/test/testset.csv 
+
+Institution:
+    Optional. Defaults to one column named 'address'
+
+Requirements:
+    shapely
+    pandas
+    xlrd (pandas Excel compatibility)
+    smartystreets_python_sdk
+"""
+import json
+import os
+from sys import argv
+from textwrap import dedent
+from shapely.geometry import shape, Point
+import logging
+import pandas as pd
+from smartystreets_python_sdk import StaticCredentials, exceptions, ClientBuilder
+from smartystreets_python_sdk.us_street import Lookup
+from smartystreets_python_sdk.us_extract import Lookup as ExtractLookup
+import config
+import click
+
+LOG = logging.getLogger(__name__)
+
+@click.command()
+@click.argument('filepath', required=True, type=click.Path(exists=True))
+@click.option('-i', '--institute', default='default', 
```

Comment:
Would this option be better as a choice option so that users know the exact acronym they need to enter?
https://click.palletsprojects.com/en/7.x/options/#choice-options

Reply:
Good find!